pyproject.toml

```diff
@@ -13,14 +13,17 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "<3.12,>=3.7.1"
-singer-sdk = { version="^0.31.1" }
+python = ">=3.8,<3.13"
```

Comment:
We might wanna go to `>=3.9` and maybe remove the upper constraint.

Reply:
https://github.com/MeltanoLabs/tap-bigquery/pull/24/commits/bc38dfaa96726c033bce35fc873c7f9265f02a0b

Can't yet remove the upper constraint as `sqlalchemy-bigquery` [does not yet support Python 3.13](https://github.com/googleapis/python-bigquery-sqlalchemy/blob/8e59f642e1192029b6adf3f938982676f0fa8284/setup.py#L122) (see https://github.com/googleapis/python-bigquery-sqlalchemy/issues/1132).